src/components/scenes/Fio/FioAddressRegisterSelectWalletScene.tsx

```diff
@@ -304,3 +317,5 @@ export const FioAddressRegisterSelectWalletScene = connect<StateProps, DispatchP
     }
   })
 )(withTheme(FioAddressRegisterSelectWallet))
+
+export const FioAddressRegisterSelectWalletScene = withWallet((props: OwnProps) => <FioAddressRegisterSelectWalletConnected {...props} />)
```

Comment:
You can simplify this to `export const FioAddressRegisterSelectWalletScene = withWallet(FioAddressRegisterSelectWalletConnected)`. I checked out the branch, and TypeScript was still happy. Likewise for the others.